### ISSUE TITLE
Error while working a few cycles

### ISSUE THREAD

**Issue by PeterBorisenko** [Disgust/oem1_usb_relay_ctrl#1]
## object.Error@(0): Access Violation

0x0041308B in @safe void std.stdio.File.detach()
0x00412E1A in D2rt6dmain211_d_run_mainUiPPaPUAAaZiZ6runAllMFZ9__lambda1MFZv
0x00412DEF in void rt.dmain2._d_run_main(int, char*_, extern (C) int function(char[][])_).runAll()
0x00412D05 in _d_run_main
0x00412924 in main
0x0043CB41 in mainCRTStartup
0x759EED5C in BaseThreadInitThunk
0x771437EB in RtlInitializeExceptionChain
0x771437BE in RtlInitializeExceptionChain
Error executing command run: Program exited with code 1

**Comment by PeterBorisenko** [comment]
progState moved to global scope
